### PR TITLE
test circleci

### PR DESCRIPTION
**Pull request by progers-SL**
WIP - testing environment variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,37 +85,44 @@ account_id=<your_account_id>
...
 # if prompted to migrate all workspaces to S3 then respond with 'yes'
-terraform init -backend-config "bucket=<your_bucket_name>-terraform-state"
+terraform init -backend-config "bucket=<app_name>-terraform-state"
 # select your workspace
 terraform workspace new <your_workspace>
 terraform apply
 ```
+
 ### Shared Code
 * Now enter the `securityanalytics-sharedcode` directory
```
Comment (blackmamo) [inline README.md]: you need to pipenv install here too

File: README.md [hunk 2]
```diff
@@ -117,74 +131,54 @@ terraform apply
...
+* Next the analytics platform needs to be deployed, enter the `securityanalytics-analyticsplatform` directory, and follow the build steps above. When deploying the infrastructure, elasticsearch takes around 10 minutes, so grab yourself a drink and wait...
+
 ### Nmap scanner
-* This task requires some Python libraries to be installed first:
-```
-pip3 install boto3
-pip3 install requests_aws4auth
-```
+
 * Enter the `securityanalytics-nmapscanner` directory. If you have both Python 2 and Python 3 installed you might need to edit the `python` references in elastic_resources/index.tf to be `python3`.
```
Comment (blackmamo) [inline README.md]: Will this work if you only have python 3 installed? If so we should change the scripts

File: terraform_backend/backend.tf [hunk 3]
```diff
@@ -6,11 +6,10 @@ variable "aws_region" {
   default = "eu-west-2" # london
 }
 
-variable "s3_app_name" {}
+variable "backend_bucket" {}
```
Comment (blackmamo) [inline terraform_backend/backend.tf]: Is this used?

File: README.md (hunk 2 above)
Comment (progers-SL) [reply README.md]: it's a user thing if they have both installed (only works on python 3) - but if they have both then they'll have to figure out if they want to edit the script (easier) or change their default path (bit more complex depending on how their install is)

File: terraform_backend/backend.tf (hunk 3 above)
Comment (progers-SL) [reply terraform_backend/backend.tf]: not any more! forgot to delete it after I baked app_name into the shared bucket

**Pull request merged**